qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -81,10 +86,21 @@ class QuteCodeActions {
 
 	private static final String UNDEFINED_NAMESPACE_SEVERITY_SETTING = "qute.validation.undefinedNamespace.severity";
 
+	private static final String APPEND_TO_TEMPLATE_EXTENSIONS = "Create template extension `%s()` in detected template extensions class";
+	
+	private static final String CREATE_TEMPLATE_EXTENSIONS = "Create template extension `%s()` in a new template extensions class";
+
+	private static final String CREATE_GETTER = "Create getter `get%s()` in %s";

```

Comment:
The second %s must be enclosed with tild. The message must be ended with dot. Do that for the other message.